make/integrations/most.Makefile

```diff
@@ -7,33 +7,66 @@ endif
 clean_unittest:
 	rm -fr build_unittest coverage
 
+# build and run all unit-tests
 test_all:
```

Comment:
Can we do `test_all: test_build_all test_run_all`

Reply:
Two targets already have the names you've stated.
The name for test_all is more appropriate,  but my idea was to make 0 changes with existing target names. The already-existing targets ones are being used by other makefile(s) , I know at least one but there might be more? The chance I will break something is higher

Reply:
Sorry, I misunderstood you.
Good idea, I will change accordingly.